dialpad/client.py

```diff
@@ -53,10 +53,9 @@ def _cursor_iterator(self, response_json, path, method, data, headers):
       data['cursor'] = response_json['cursor']
       response = self._raw_request(path, method, data, headers)
       response.raise_for_status()
-      if response != []:
-        response_json = response.json()
-        for i in response_json['items']:
-          yield i
+      response_json = response.json() or dict()
+      for i in response_json['items']:
```

Comment:
accessing the key 'items' this way can throw a `KeyError` if the `response_json` dictionary is empty. `response_json.get('items', [])` will fail gracefully instead.